google-beta/resource_composer_environment_test.go

```diff
@@ -280,6 +314,27 @@ resource "google_composer_environment" "test" {
 `, name)
 }
 
+func testAccComposerEnvironment_private(name string) string {
+	return fmt.Sprintf(`
+resource "google_composer_environment" "test" {
+	name           = "%s"
+	region         = "us-central1"
+
+	config {
+		node_config {
+			zone = "us-central1-a"
+			ip_allocation_policy {
+				use_ip_aliases = true
+			}
+		}
+		private_environment_config {
+			enable_private_endpoint = true
```

Comment:
Was there an issue with testing the other fields (like range overlap)? Probably would be a good idea to test actually setting the other fields for cidr ranges